lib/msf/java/rmi/client.rb

```diff
@@ -129,7 +146,7 @@ def safe_get_once(nsock = sock)
             res = nil
           end
 
-          until res.nil? || res.length < 1448
```

Comment:
already discussed in person, but for people who wasn't here :) I was expecting to deal just with IP fragmentation. And of course, I was definitely wrong :\